docs/keypoints_convention.md

```diff
@@ -1,52 +1,68 @@
-## Keypoints convention
+# Keypoints convention
 
-### Overview
+## Overview
 
 Our convention tries to consolidate the different keypoints definition across various
 commonly used datasets. Due to differences in data-labelling procedures, keypoints across datasets with the same name might not map to semantically similar locations on the human body. Conversely, keypoints with different names might correspond to the same location on the human body. To unify the different keypoints correspondences across datasets, we adopted the `human_data` convention
 as the base convention for converting and storing our keypoints.
 
-### How to use
+## How to use
 
-#### Converting between conventions
+### Converting between conventions
 
 Keypoints can be converted between different conventions easily using the `convert_kps` function.
 
-To convert a `agora` keypoints to `coco` convention, specify the source and
+To convert a `human_data` keypoints to `coco` convention, specify the source and
 destination convention for conversion.
 
 ```python
 from mmhuman3d.core.conventions.keypoints_mapping import convert_kps
 
-keypoints_agora = np.zeros((100, 127, 3))
-keypoints_coco, mask = convert_kps(keypoints_agora, src='agora', dst='coco')
+keypoints_human_data = np.zeros((100, 190, 3))
+keypoints_coco, mask = convert_kps(keypoints_human_data, src='human_data', dst='coco')
```

Comment:
Maybe we need to highlight the difference between `mask` and `confidence`. What are their purposes and how are they generated/obtained. And indicate the shape of `keypoints` already includes `confidence`.

Reply:
As discussed, we should modify this in the next PR when `convert_kps` is changed.